src/open_clip/transformer.py

```diff
@@ -465,6 +465,9 @@ def forward(self, x: torch.Tensor):
         x = self.transformer(x)
         x = x.permute(1, 0, 2)  # LND -> NLD
 
+        if output_tokens:
+            return x
+        
         if self.global_average_pool:
             x = x.mean(dim=1)
```

Comment:
I'm wondering if this can be done after the ln post

if yes, then it will make it possible to do the ln post only here and not in coca